### DISCUSSION TITLE
Is there anyway to jump back to your original position just before leaping?

### DISCUSSION THREAD

**Discussion by ghost** [ggandor/leap.nvim#232] [Q&A] [answered]
Is there a command/keybinding to jump back to where your cursor was just before you used leap?

So if I am on line 32, i jump to something on line 58 to quickly change something and then I want to go back to where my cursor was before I jumped how can i do that?

**Comment by ggandor** [comment, marked as answer]
`ctrl-o` is what you're looking for (`:help CTRL-O`, `:help jumplist`).

**Reply by ghost** [reply]
Perfect thank you so much!
